StarkInfra/StarkInfra/IndividualIdentity/IndividualIdentity.cs

```diff
@@ -0,0 +1,356 @@
+        ///     <item>before [DateTime, default null] date filter for objects created only before specified date. ex: DateTime(2020, 3, 10)</item>
+        ///     <item>status [list of string, default null]: filter for status of retrieved objects. ex: "created", "canceled", "processing", "failed", "success"</item>
+        ///     <item>tags [list of strings, default null]: list of tags to filter retrieved objects. ex: new List<string>{ "tony", "stark" }</item>
+        ///     <item>ids [list of strings, default null]: list of ids to filter retrieved objects. ex: new List<string>{ "5656565656565656", "4545454545454545" }</item>
+        ///     <item>user [Organization/Project object, default null]: Organization or Project object. Not necessary if StarkInfra.Settings.User was set before function call</item>
+        /// </list>
+        /// <br/>
+        /// Return:
+        /// <list>
+        ///     <item>list of IndividualIdentity objects with updated attributes</item>
+        ///     <item>cursor to retrieve the next page of IndividualIdentity objects</item>
+        /// </list>
+        /// </summary>
+        public static (List<IndividualIdentity> page, string pageCursor) Page(
+            string cursor = null, int? limit = null, DateTime? after = null, DateTime? before = null, 
+            List<string> status = null, List<string> tags = null,List<string> ids = null, User user = null
+        ) {
+            (string resourceName, Api.ResourceMaker resourceMaker) = Resource();
+            (List<SubResource> page, string pageCursor) = Rest.GetPage(
+                resourceName: resourceName,
+                resourceMaker: resourceMaker,
+                query: new Dictionary<string, object> {
+                    { "cursor", cursor },
+                    { "limit", limit },
+                    { "after", after },
+                    { "before", before },
+                    { "tags", tags },
+                    { "ids", ids }
+                },
+                user: user
+            );
+            List<IndividualIdentity> identities = new List<IndividualIdentity>();
+            foreach (SubResource subResource in page)
+            {
+                identities.Add(subResource as IndividualIdentity);
+            }
+            return (identities, pageCursor);
+        }
+
+        /// <summary>
+        /// Update IndividualIdentity entity
+        /// <br/>
+        /// Update an IndividualIdentity by passing its id.
+        /// <br/>
+        /// Parameters(required):
+        /// <list>
+        ///     <item>id[string]: IndividualIdentity id. ex: "5656565656565656"</item>
+        /// </list>
+        /// <br/>
+        /// Parameters (optional):
+        /// <list>
+        ///     <item>status [string, default null]: You may block the IndividualIdentity by passing "blocked" or activate by passing "active" in the status</item>
```

Comment:
You may send IndividualDocuments to validation by passing 'processing' in the status